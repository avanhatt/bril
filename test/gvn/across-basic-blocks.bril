void main(x : int, y : int) {
entry:
  z1 : int = add x y;
  jmp block;
block:
  z2 : int = add x y;
  ret;
}