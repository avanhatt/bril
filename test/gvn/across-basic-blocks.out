void main(x : int, y : int) {
entry:
  z11: int = add x y;
  jmp block;
block:
  print z11;
  print z11;
  ret ;
}
